deltachat-ios/Controller/QrViewController.swift

```diff
@@ -115,7 +126,7 @@ class QrViewContentView: UIView {
 
     init(qrCode: String?, hint: String) {
         super.init(frame: .zero)
-        hintLabel.text = hint
+        hintText = hint
```

Comment:
why this change? hintText is used only internally if I understand it correctly and therefore it adds only boilerplate code without benefits.